ocrd/ocrd/decorators/ocrd_cli_options.py

```diff
@@ -19,28 +19,28 @@ def cli(mets_url):
     """
     # XXX Note that the `--help` output is statically generate_processor_help
     params = [
-        option('-m', '--mets', help="METS to process", default="mets.xml"),
-        option('-w', '--working-dir', help="Working Directory"),
+        option('-m', '--mets', default="mets.xml"),
+        option('-w', '--working-dir'),
         # TODO OCR-D/core#274
-        # option('-I', '--input-file-grp', help='File group(s) used as input. **required**'),
-        # option('-O', '--output-file-grp', help='File group(s) used as output. **required**'),
-        option('-I', '--input-file-grp', help='File group(s) used as input.', default='INPUT'),
-        option('-O', '--output-file-grp', help='File group(s) used as output.', default='OUTPUT'),
-        option('-g', '--page-id', help="ID(s) of the pages to process"),
-        option('--overwrite', help="Overwrite the output file group or a page range (--page-id)", is_flag=True, default=False),
-        option('--queue', help="The URL of the Queue Server, format: username:password@host:port/vhost", type=QueueServerParamType()),
-        option('--database', help="The URL of the MongoDB, format: mongodb://host:port", type=DatabaseParamType()),
-        option('-C', '--show-resource', help='Dump the content of processor resource RESNAME', metavar='RESNAME'),
-        option('-L', '--list-resources', is_flag=True, default=False, help='List names of processor resources'),
+        # option('-I', '--input-file-grp', required=True),
+        # option('-O', '--output-file-grp', required=True),
+        option('-I', '--input-file-grp', default='INPUT'),
+        option('-O', '--output-file-grp', default='OUTPUT'),
+        option('-g', '--page-id'),
+        option('--overwrite', is_flag=True, default=False),
+        option('--profile', is_flag=True, default=False),
+        option('--profile-file', type=Path(dir_okay=False, writable=True)),
```

Comment:
That's probably wise since we don't use the click documentation features anyway.

Reply:
That was the idea. We _could_ try to reuse these in generate_processor_help, but then we would not be able to do more fancy stuff there, like grouping the options.

Reply:
I've run into this myself more than once, that I forget the `--help` output is just a string and not generated from the click documentation.